Ms2Database/DBClasses/Ms2DBContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ms2Database.DBClasses
+{
+    public class Ms2DBContext : DbContext
+    {
+        public Ms2DBContext() : base("name=DatabaseConnectionString")
+        {
+            //Database.SetInitializer<Ms2DBContext>(new DropCreateDatabaseAlways<Ms2DBContext>());
```

Comment:
Why is this commented?

Reply:
It re-creates the DB each time ms2database is ran. I have it commented out so it doesn't do that for everyone that uses this. It would be too much work for me to put it back in whenever I need to test database structure stuff.